rest_docs/rest-catalog-open-api.yaml

```diff
@@ -564,12 +648,21 @@ components:
           example: '{ "owner": "Hank Bendickson" }'
           default: '{ }'
 
-    Namespace:
-      description: Reference to one or more levels of a namespace
-      type: array
-      items:
-        type: string
-      example: [ "accounting", "tax" ]
+    UpdateNamespacePropertiesRequest:
+      type: object
+      properties:
+        removals:
+          type: array
+          uniqueItems: true
+          items:
+            type: string
+          example: '[ "department", "access_group" ]'
```

Comment:
Good question for @kbendick!

Reply:
The extra single quotes are allowed from my experience. It will simply parse that whole thing as a JSON string (at least the various UI editors I've tried have allowed that).

Reply:
Actually, another CLI tool I put it in called `widdershins` didn't like it. They should be removed.